src/api/Misc.ts

```diff
@@ -10,8 +10,25 @@ export interface HomepageLink {
 	badge?: string | null;
 }
 
+export interface BulkData {
+	id: string;
+	type: string;
+  name: string;
+  description: string;
+  permalink_uri: string;
+  created_at: string;
+  updated_at: string;
+  compressed_size: number;
+  content_type: string;
+  content_encoding: string;
+}
```

Comment:
Fix indent & add object discriminator
```suggestion
export interface BulkData {
	object: "bulk_data";

	id: string;
	type: string;
	name: string;
	description: string;
	permalink_uri: string;
	created_at: string;
	updated_at: string;
	compressed_size: number;
	content_type: string;
	content_encoding: string;
}
```